package.json

```diff
@@ -25,5 +30,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "prettier": {
```

Comment:
Is there no way to avoid this config here? Can not be moved any other place? :cry:

Reply:
there are 3 options how to save config: https://prettier.io/docs/en/configuration.html
I prefer in package.json. But if you like .rc file, I'll change it